src/scopeHandler.js

```diff
@@ -0,0 +1,13 @@
+const qs = require('querystring')
+
+const scopes = {
+  decode: function (scopesString) {
+    if (Array.isArray(scopesString)) {
+      return scopesString
+    }
+    const decodedScopes = qs.parse(`scope=${scopesString}`)
+    return decodedScopes.scope.split(' ')
+  }
+}
+
+module.exports = scopes
```

Comment:
You're right, i work mainly with typescript / ES6 Modules these days where default exports like the above can cause issues so we name them all. 

Here its fine though.